.circleci/run_coverage.sh

```diff
@@ -1,20 +1,17 @@
 #!/bin/bash
-
-ninja all
-ninja glow_coverage
-
+cd build 
```

Comment:
why do we need to add this? (what has changed)

Reply:
i've cleaned it